athena-elasticsearch/src/test/java/com/amazonaws/athena/connectors/elasticsearch/integ/ElasticsearchIntegTest.java

```diff
@@ -184,6 +186,33 @@ protected void setUpTableData()
                             "David Gyasi", "Michael Caine", "Jessica Chastain", "Matt Damon", "Casey Affleck")
             ));
         }
+        try (ElasticsearchIndexUtils indexUtils = new ElasticsearchIndexUtils(domainName, TEST_DATATYPES_TABLE_NAME))
+        {
+            Map<String, Object> item = new ImmutableMap.Builder<String, Object>()
```

Comment:
Not sure what our guidelines are, but if they are to prefer Java standard library stuff over Guava, you may want to use Map.of() or Map.ofEntries instead of this Guava builder.

https://docs.oracle.com/javase/9/docs/api/java/util/Map.html#immutable

Reply:
We use Guava pretty much everywhere, so I've kept that pattern for now. If we want to remove Guava, we'd do it everywhere as a separate PR.